Source/Scene/ModelExperimental/PrimitiveRenderResources.js

```diff
@@ -112,14 +112,14 @@ export default function PrimitiveRenderResources(
   this.alphaOptions = clone(nodeRenderResources.alphaOptions);
 
   /**
-   * The model space transform for this primitive. This is cloned from the
-   * node render resources as the primitive may further modify it
+   * The scene graph transform for this primitive. This is cloned from the
+   * node render resources as the primitive may further modify it.
    *
    * @type {Matrix4}
    *
    * @private
    */
-  this.transform = nodeRenderResources.runtimeNode.transform.clone();
+  this.transform = Matrix4.clone(nodeRenderResources.transform);
```

Comment:
Not that I know of. Could you try this quick experiment: change this to `this.transform = nodeRenderResources.transform;` (i.e. copy the pointer instead of clone) and see if anything breaks? If it works, I would update this (and the documentation comment)